prowler/providers/aws/services/cognito/cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in/cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in.py

```diff
@@ -0,0 +1,38 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.cognito.cognito_idp_client import cognito_idp_client
+
+
+class cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in(
+    Check
+):
+    def execute(self):
+        findings = []
+        for pool in cognito_idp_client.user_pools.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = pool.region
+            report.resource_name = pool.name
+            report.resource_id = pool.id
+            report.resource_arn = pool.arn
+            if (
+                pool.advanced_security_mode == "ENFORCED"
+                and pool.risk_configuration.account_takeover_risk_configuration.get(
```

Comment:
I think it'd be better to map this into an object.

Reply:
Yes, good point